apps/extensions/src/content/panel/components/containers/dashboardLayout/elements/SidebarMenuItems/SidebarMenuItem.tsx

```
@@ -17,21 +20,127 @@ const SidebarMenuItem: React.FC<ISidebarMenuItemProps> = ({
+            )}
+          >
+            {active && <div className={styles.activeIndicator} />}
+
+            <div className={styles.contentOuterWrapper}>
+              <div
+                className={classNames(
+                  styles.contentWrapper,
+                  active && styles.active,
+                )}
+              >
+                <button onClick={onClick} className="tw-flex">
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)